src/App.js

```diff
@@ -21,6 +22,7 @@ function App() {
           <Route path='/admin-app/groups' exact component={GroupsPage} />
           <Route path='/admin-app/students/:id' exact component={StudentDetailPage} />
           <Route path={['/admin-app/students', '/']} exact component={StudentsPage} />
+          <Route path={['/admin-app/payments', '/']} exact component={PaymentsPage} />
```

Comment:
```suggestion
          <Route path='/admin-app/payments',  exact component={PaymentsPage} />
```
Сейчас в качестве главной старицы - страница "студенты"